src/Adapter/MSTest.CoreAdapter/Extensions/MethodInfoExtensions.cs

```diff
@@ -143,6 +145,14 @@ internal static string GetAsyncTypeName(this MethodInfo method)
         /// </param>
         internal static void InvokeAsSynchronousTask(this MethodInfo methodInfo, object classInstance, params object[] parameters)
         {
+            if (methodInfo.GetParameters() != null && parameters == null)
+            {
+                string errorMessage = string.Format(
```

Comment:
ResourceManager takes care of culture, do we need this formating?

Reply:
Validated on a DEU machine. ResourceManager working expectedly. 👍